src/utils/signaling.js

```diff
@@ -1066,7 +1066,7 @@ Signaling.Standalone.prototype.helloResponseReceived = function(data) {
 
 		// TODO(fancycode): How should this be handled better?
 		const url = this._getBackendUrl()
-		console.error('Could not connect to server using backend url ' + url, data)
+		console.error('Could not connect to server using backend url %s %s', url, data)
```

Comment:
data is fine being a string?

Reply:
Error() and Date() constructors are fine, but object would be `[object Object]`. Need to use `%o` or `%O`:
<img width="475" alt="image" src="https://github.com/user-attachments/assets/a7c0b196-bbd3-414a-bcf7-b29a1cd9a041" />